all/build.gradle.kts

```diff
@@ -1,24 +1,19 @@
-plugins {
-  id("otel.java-conventions")
-}
+plugins { id("otel.java-conventions") }
```

Comment:
Don't think Gradle DSL looks good with all the one-liners (which would look good for normal lambda invocations in general, but not DSL)

Reply:
I have no issue with that. I would much prefer a general kts/Kotlin formatter over no formatter, even if it sometimes leads to sub-optimal results.